pkg/controller/networkpolicy/crd_utils.go

```diff
@@ -108,6 +110,36 @@ func (n *NetworkPolicyController) toAntreaPeerForCRD(peers []secv1alpha1.Network
 	return &controlplane.NetworkPolicyPeer{AddressGroups: addressGroups, IPBlocks: ipBlocks}
 }
 
+// createAppliedToGroupForClusterGroupCRD creates an AppliedToGroup object corresponding to a
+// ClusterGroup spec. If the AppliedToGroup already exists, it returns the key
+// otherwise it copies the ClusterGroup CRD contents to an AppliedToGroup resource and returns
+// its key.
+func (n *NetworkPolicyController) createAppliedToGroupForClusterGroupCRD(cg *v1alpha2.ClusterGroup) string {
+	// Find the internal Group corresponding to this ClusterGroup
+	igKey := internalGroupKeyFunc(cg)
+	ig, found, _ := n.internalGroupStore.Get(igKey)
+	if !found {
+		klog.V(2).Infof("Internal group %s not found.", igKey)
+		return ""
```

Comment:
Same question as the one in another PR. What if the internal group hasn't been created but will be created?